src/grip/specifications/grip.md

```diff
@@ -0,0 +1,31 @@
+# Grip
+
+Use it to indicate that a part of the screen is reactive to slide touch events.
+
+<!-- STORY -->
+
+## Usage
+
+`TextComponents` are exported individually.
+
+```js
+import Grip from '@blablacar/ui-library/build/grip'
+;<Grip
```

Comment:
I have the same issue on formating :(
Remove the `;` before `<Grip...`

Reply:
Thanks! Done ;)